administrator-documentation/moderne-platform/how-to-guides/dev-center.md

```diff
@@ -0,0 +1,329 @@
+
+If you have not configured the Moderne agent with Maven repository access before, please follow [the instructions in our Moderne agent Maven configuration documentation](/administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-an-agent-with-maven-repository-access.md) to add **one** entry with recipe source set to `true`.
+
+If you have already configured the Moderne agent with Maven repository access, then you need to ensure that only _one_ has the configuration of `MODERNE_AGENT_MAVEN_{index}_RECIPESOURCE` / `moderne.agent.maven[{index}].recipeSource` set to `true`. 
+
+If you have multiple locations where recipes are stored, you will need to create a virtual repository that wraps all of the locations where recipes can be stored. You will also need to ensure that the virtual repository points to the following four repositories (alongside the other repositories where recipe artifacts are stored):
+
+1. `https://oss.sonatype.org/content/repositories/snapshots/`
+2. `https://s01.oss.sonatype.org/content/repositories/snapshots/`
+3. `https://repo.maven.apache.org/maven2`
+4. `https://repo1.maven.org/maven2/` 
+
+## Step 1: Ensure you have a `DevCenterDataFetcher` class
+
+_This step only applies if you used the [Moderne Organizations service template](https://github.com/moderneinc/moderne-organizations). If you made your own, please jump to [step 3](#step-3-create-and-configure-the-devcenter)._
+
+If you've created an Organizations service prior to March 2024, you will need to copy the [new DevCenterDataFetcher file](TODO LINK WHEN READY) to your Organizations service repository. It will go in the same location as the other source classes such as [Application.java](https://github.com/moderneinc/moderne-organizations/blob/main/src/main/java/io/moderne/organizations/Application.java).
+
+If you've created an Organizations service after March 2024, please ensure that you have a `DevCenterDataFetcher.java` file in your Organizations service repository before moving on to step 2.
+
+## Step 2: Ensure you have an up-to-date `moderne-organizations.graphqls` schema
+
+Similar to the previous step, please double-check your [moderne-organizations.graphqls file](https://github.com/moderneinc/moderne-organizations/blob/main/src/main/resources/schema/moderne-organizations.graphqls) and ensure that there is a `devCenter` field in the `Organization` object:
+
+```graphql
+type Organization {
+    id: ID!
+    name: String!
+
+    """
+    Ordered list of commit options as they should appear in the UI.
+    """
+    commitOptions: [CommitOption!]!
+
+    parent: Organization
+
+    devCenter: DevCenter
+}
+```
+
+The [DevCenter object](TODO LINK WHEN READY) is the schema you need to follow in the below step to configure your DevCenter.
+
+## Step 3: Create and configure the DevCenter
```

Comment:
They will also need to run the graphql generate tasks to get the types, e.g. [DevCenter](https://github.com/moderneinc/moderne-organizations/pull/61/files#diff-5e9d47c60e3585a5f33205fb5367fa474776265d832cf0d1acab8dec9278550a)

Reply:
`./gradlew generateGraphqlJava copyGeneratedGraphql` will generate all the types they need.

Of course, only applicable if they're using our template.